core/editable.js

```diff
@@ -2273,7 +2288,7 @@
 				wrapper = doc.createText( '{cke-peak}' ),
 				limit = that.inlineStylesRoot.getParent();
 
-			while ( !element.equals( limit ) ) {
+			while ( element && !element.equals( limit ) ) {
```

Comment:
What is the reason for this change? If it's required, could you cover it with tests?

Reply:
Without this conditional sometimes I had an error when clicking randomly clicking in editor and then using one of test buttons. I couldn't reproduce some weird selection which caused the issue.

Reply:
That doesn't look like a good reason to change this code without tests. I would suggest trying to reproduce the issue or if it's no longer reproducible, remove the condition. It looks like an element doesn't have a parent at some point which means it's not attached to the DOM. It may be caused by a different bug and hiding it will make harder to find a real problem in the future.

Are you able to reproduce the issue without your changes? Please, try to find a real problem, we cannot leave it untested.

Reply:
I see your point. I can't reproduce mentioned issue, so I'm re moving the condition.